CHANGES.md

```diff
@@ -3,8 +3,11 @@ Change Log
 
 ### MobX Development
 
-#### next release (8.0.0-alpha.59)
+#### next release (8.0.0-alpha.60)
```

Comment:
Same as other PR: We don't need this, it's a bit confusing - the next release is .59

Reply:
fixed